### PR TITLE
[SPC-92] Add functionality to track interactions with contracts

### PR DESCRIPTION
**Pull request by lucas-marc**
## Description
Added wasm module to process wasm messages. This allows to load contracts information to the database. In this version, the the imports were modified to use the ibc-go v2 version instead of the v3 one, as it was trying to import the v2 and the v3 causing a `panic: error with code 2 is already registered: "invalid identifier"` error. After compiling, the application should be able to start and observed multiple rows being added to the following tables: 
- block
- codes
- contracts
- pre_commit
- tokens
- transaction
- validator

Also, the package name was changed to use Source-protocol/juno

## Related tasks
- *[SPC-92](https://thinkanddev.atlassian.net/browse/SPC-92)*: Add functionality to track interactions with contracts

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -44,6 +44,21 @@ type Database interface {
 
 	// Close closes the connection to the database
 	Close()
+
+	// WASMX extras
+	// Tokens
+    SaveToken(token types.Token) error 
+
```
Comment (gonzalo-smtv) [inline database/database.go:51]: Litle indentation error

File: README.md [hunk 2]
```diff
@@ -6,45 +6,50 @@
...
-| Cosmos SDK Version | Juno branch |
-| :----------------: | :---------: | 
-| `v0.37.x` | `cosmos-v0.37.x` |
-| `v0.38.x` | `cosmos-v0.38.x` |
-| `v0.39.x` | `cosmos-v0.39.x` |
+|               Cosmos SDK Version                |   Juno branch    |
+| :---------------------------------------------: | :--------------: |
+|                    `v0.37.x`                    | `cosmos-v0.37.x` |
+|                    `v0.38.x`                    | `cosmos-v0.38.x` |
+|                    `v0.39.x`                    | `cosmos-v0.39.x` |
```
Comment (gonzalo-smtv) [inline README.md:44]: 💯

**Review by gonzalo-lopezlabs — APPROVED** [review]
The real merge! 🧩🧩

File: database/database.go (hunk 1 above)
Comment (lucas-marc) [reply database/database.go:51]: Fixed

**Pull request merged**